chart/kubeapps/templates/apprepository-rbac.yaml

```diff
@@ -142,4 +142,19 @@ rules:
       - secrets
     verbs:
       - '*'
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: "kubeapps:{{ .Release.Namespace }}:apprepositories-refresh"
+  labels:{{ include "kubeapps.extraAppLabels" . | nindent 4 }}
+  app: {{ template "kubeapps.apprepository.fullname" . }}
```

Comment:
We missed the indentation here - `app:` is a label, not a field on metadata - which caused CI to fail, but we currently have false positives so it makes a genuine failure easier to ignore. We should fix that :) Anyway, I've fixed it as part of #2162 .

Reply:
My bad! I didn't notice it! Thanks for pointing out and fix it!!!